Source: @pullmoll
Upstream: no
Reason: Enable subdir-objects and strip openssl version check which fails for libressl

--- a/configure.ac	2018-08-16 01:06:21.000000000 +0200
+++ b/configure.ac	2020-01-27 15:02:36.946922663 +0100
@@ -58,7 +58,7 @@
 dnl	AM_INIT_AUTOMAKE([1.8])
 dnl	AM_INIT_AUTOMAKE([1.9 tar-ustar])
 dnl
-AM_INIT_AUTOMAKE([1.9 tar-ustar])
+AM_INIT_AUTOMAKE([1.9 tar-ustar subdir-objects])
 
 AM_MAINTAINER_MODE
 
@@ -867,32 +867,6 @@
 	;;
 esac
 
-AC_MSG_CHECKING([OpenSSL version])
-AC_TRY_COMPILE([
-#include <openssl/opensslv.h>
-], [
-#if OPENSSL_VERSION_NUMBER < 0x0090801fL
-#error OpenSSL version too old 
-#endif
-    ],[
-      AC_MSG_RESULT([>= 0.9.8e])
-    ],[
-      AC_MSG_ERROR([OpenSSL version 0.9.8e or greater is required.])
-    ])
-
-AC_MSG_CHECKING([OpenSSL version])
-AC_TRY_COMPILE([
-#include <openssl/opensslv.h>
-],[
-#if OPENSSL_VERSION_NUMBER >= 0x1010000fL
-#error OpenSSL version too new
-#endif
-    ],[
-      AC_MSG_RESULT([< 1.1.0])
-    ],[
-      AC_MSG_ERROR([OpenSSL version 1.1.0 or greater is currently not supported.])
-    ])
-
 AC_MSG_CHECKING([usability of the OpenSSL header files and library in ${openssl_dir}])
 AC_TRY_RUN([
 #include <openssl/rsa.h>
